File: C/tabuada.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int N, resultado;

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d", &N);

    for (i = 1; i <= 10; i++) {
          resultado = N * i;
          printf("%d x %d = %d\n", N, i, resultado);
    }

    return 0;
}

File: java/tabuada.java
import java.util.Locale;
import java.util.Scanner;

public class tabuada {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		
		System.out.print("Deseja a tabuada para qual valor: ");
		n = sc.nextInt();
		
		for (int i = 1; i <= 10; i++) {
			System.out.println(n + " x " + i + " = " + n * i);
		}
		
		sc.close();
	}

}
